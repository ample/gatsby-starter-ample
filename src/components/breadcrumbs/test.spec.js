import React from "react"
import renderer from "react-test-renderer"

import { component as Breadcrumbs, fixtures } from "."

describe("Breadcrumbs", () => {
  it("renders correctly", () => {
    const tree = renderer.create(<Breadcrumbs {...fixtures.default} />).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
