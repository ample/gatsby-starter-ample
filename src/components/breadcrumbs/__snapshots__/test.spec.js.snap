// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Breadcrumbs renders correctly 1`] = `
<ul
  className="breadcrumbs"
>
  <li
    data-separator="/"
  >
    <a
      href="#"
      rel="noopener"
      target="_blank"
    >
      Lorem
    </a>
  </li>
  <li
    data-separator="/"
  >
    <a
      href="#"
      rel="noopener"
      target="_blank"
    >
      ipsum
    </a>
  </li>
  <li
    data-separator="/"
  >
    <a
      href="#"
      rel="noopener"
      target="_blank"
    >
      dolor
    </a>
  </li>
  <li
    data-separator="/"
  >
    <a
      href="#"
      rel="noopener"
      target="_blank"
    >
      sit
    </a>
  </li>
  <li
    data-separator="/"
  >
    <a
      href="#"
      rel="noopener"
      target="_blank"
    >
      amet
    </a>
  </li>
</ul>
`;
